src/components/Attachment/Attachment.tsx

```diff
@@ -56,9 +62,9 @@ const Attachment = ({ fileName }) => {
     });
   }, [extractedFile, fileName, isMounted]);
 
-  if (error) return error.toString();
+  if (error) return <p>error.toString()</p>;
```

Comment:
I wrapped the error in JSX, because we were returning a mix of JSX and raw text, and TS warned me it wasn't a valid React component.

Reply:
Returning JSX is not the problem and I think it's a good idea, but you should have used `<p>{error.toString()}</p>` with the curly brakets otherwise it will litterally return the string `error.toString()` 😉

Reply:
Ah LOL didn't catch that..

Reply:
I'll push a fix tomorrow!

Reply:
No need, I already took care of it, I'm about to push my changes